src/components/AppNavigationItem/AppNavigationItem.vue

```diff
@@ -19,92 +19,71 @@
   - along with this program. If not, see <http://www.gnu.org/licenses/>.
   -
   -->
+<docs>
+
+### Props
```

Comment:
Props should be documented by JSDoc comment blocks in the script part, see https://vue-styleguidist.github.io/docs/Documenting.html#code-comments

Reply:
Sure, will do :)